react/features/prejoin/actions.js

```diff
@@ -121,6 +280,22 @@ export function joinConferenceWithoutAudio() {
     };
 }
 
+/**
+ * Opens an external page with all the dial in numbers.
+ *
+ * @returns {Function}
+ */
+export function openDialInPage() {
+    return function(dispatch: Function, getState: Function) {
+        const state = getState();
+        const locationURL = state['features/base/connection'].locationURL;
+        const roomName = getRoomName(state);
+        const dialInPage = getDialInfoPageURL(roomName, locationURL);
+
+        window.open(dialInPage, '_blank', 'noopener');
```

Comment:
Use `openURLInBrowser` from base/utils. You may need to extend it to support the target.

Reply:
Done.